app/wizards/schools/register_mentor_wizard/check_answers_step.rb

```diff
@@ -12,7 +12,12 @@ def previous_step
     private
 
       def persist
-        AssignMentor.new(ect:, mentor: mentor.register!).assign!
+        ActiveRecord::Base.transaction do
+          AssignMentor.new(ect:, mentor: mentor.register!).assign!
+        end
```

Comment:
There's a transaction in the `.assign!` method, I don't think we need one here too?